JsonSchema.Generation/Generators/ObjectSchemaGenerator.cs

```diff
@@ -184,24 +181,24 @@ private static IEnumerable<IConditionAttribute> ExpandEnumConditions(IConditionA
 				var values = Enum.GetValues(memberType);
 				foreach (var value in values)
 				{
-					yield return new IfAttribute(ifEnumAttribute.PropertyName, ifEnumAttribute.UseNumbers ? value : value.ToString(), value);
+					yield return (new IfAttribute(ifEnumAttribute.PropertyName, ifEnumAttribute.UseNumbers ? value : value.ToString(), value), member);
```

Comment:
it's a technical consequence of the unification.

But, are you fine with doing it? I didn't dive into the whole condition thing yet, just modified it _to the best of my belief_.